deep-map/index.d.ts

```diff
@@ -0,0 +1,65 @@
+import { WritableAtom } from '../atom/index.js'
+import { AllKeys, GetPath } from './path.js'
+
+export * from './path.js'
+
+type Listener<T extends Record<string, unknown>> = (
+  listener: (value: T, changedKey: undefined | AllKeys<T>) => void
+) => () => void
+
+export type DeepMapStore<T extends Record<string, unknown>> = Omit<
```

Comment:
Can we pass `deepMap` to argument for `atom` after omitting and replacing this methods?

Reply:
Can you explain it once again? I didn't follow.

Reply:
Does this code work:

```ts
import { deepMap, ReadableAtom } from 'nanostore'

function someStoreFunc (store: ReadableAtom) {
  …
}

someStoreFunc(deepMap())
```

Reply:
Yeah, sure, it will. `ReadableAtom` has a very simple signature.